script.js

```diff
@@ -2,11 +2,164 @@
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  displayResults();
+  selectEpisode();
 }
 
-function makePageForEpisodes(episodeList) {
+function makePageForEpisodes(episodesList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+
+  let ourSection = document.createElement("section");
+  ourSection.classList.add("allEpisodes")
+  rootElem.appendChild(ourSection);
+
+  for (let episodeIndex = 0; episodeIndex < episodesList.length; episodeIndex++) {
+    let episodeContent = document.createElement("article");
+    episodeContent.classList.add("episodeWrapper");                                               //Can I make the 'class' name same as the variable name since we can not use the variable name in CSS
```

Comment:
If you're asking whether you can have a Javascript variable called `episodeContent` and also have a CSS class name called `episodeContent`, the answer is yes.

CSS does not know about Javascript at all and has it's own scope, so you can can reuse that name.

Reply:
@louisechow that is good to know. Thank you